jax/_src/numpy/lax_numpy.py

```diff
@@ -3156,7 +3156,8 @@ def vdot(a, b, *, precision=None):
 
 
 @util._wraps(np.tensordot, lax_description=_PRECISION_DOC)
-def tensordot(a, b, axes=2, *, precision=None):
+def tensordot(a, b, axes: Union[int, Sequence[int], Sequence[Sequence[int]]] = 2,
```

Comment:
As long as we're here, could we annotate `a` and `b` with `ArrayLike` and the return type as `-> Array`?